mockup/patterns/relateditems/pattern.js

```diff
@@ -252,11 +252,18 @@ define([
             v: this.options.rootPath + this.currentPath + (this.browsing ? '::1' : '')
           });
 
+          var sort_on = this.options.sortOn;
+          var sort_order = sort_on ? this.options.sortOrder : null;
+          if (this.browsing && sort_on === null) {
+            sort_on = 'getObjPositionInPath';
```

Comment:
I've overseen this. should read ``getObjPositionInParent`` instead ``getObjPositionInPath``.

Reply:
Gah, right! Thanks for fixing.